internal/manifest/manifest.go

```diff
@@ -131,13 +140,14 @@ func ExecuteDeployments(ctx context.Context, manifests []*Manifest, client mazza
 			return err
 		}
 
-		fmt.Println("Contract Deployed:transaction id:", id)
+		fmt.Println("contract deployed:transaction id:", id)
 		receipt, err := pollForReceipt(m.Channel.Id, fmt.Sprintf("%b", id), client)
```

Comment:
This is another spot where the Transaction ID should be converted to hex for print:
```
txId := hex.EncodeToString(id[:])
```

Then txID can be used in the pollForReceipt function.  The fmt.Sprintf is setting an invalid Transaction ID currently.

Reply:
It looks like `fmt.Sprintf("%b", id)` is still being used for some of the pollForReceipt calls which make it use an invalid Transaction ID.  Can these be updated?